rest/sgcollect.go

```diff
@@ -53,7 +53,7 @@ type sgCollect struct {
 }
 
 // Start will attempt to start sgcollect_info, if another is not already running.
-func (sg *sgCollect) Start(ctxSerialNumber uint64, zipFilename string, params sgCollectOptions) error {
+func (sg *sgCollect) Start(config *ServerConfig, ctxSerialNumber uint64, zipFilename string, params sgCollectOptions) error {
```

Comment:
If all we're using from config is LogFilePath, we should just pass that in, instead of the entire config.

Reply:
Sure, that makes sense. Changes included in latest rveision.